feature-libs/user/profile/components/update-password/update-password-component.service.spec.ts

```diff
@@ -12,22 +14,35 @@ import { of } from 'rxjs';
 import { UpdatePasswordComponentService } from './update-password-component.service';
 import createSpy = jasmine.createSpy;
 
-class MockUserPasswordService implements Partial<UserPasswordFacade> {
+class MockUserPasswordFacade implements Partial<UserPasswordFacade> {
```

Comment:
I think the name should be changed to MockUserUpdatePasswordService since that is what is being tested, what do you think?